finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,88 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Kyra";
+var specialAbility = "laughter";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Hello " + heroName + "!";
+var catchsphrase = `${specialAbility} is what we're after!`;
 
 // Declare two variables - power AND energy - set to integers
+power = 11;
+energy = 33;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = power * 500;
+var faullEnergy = energy * 150;
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Syndrome", "Randall", "Darla"];
+var sidekicks = ["Samwise Gamgee", "Chewbacca", "Donkey"];
 
 // Print the first sidekick to your console
+console.log(sidekicks[0]);
 
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("Lots-o");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
 
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift();
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
-// Your function should include an if/else statement that meets the following criteria
-//   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
-//   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
-//   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  dangerLevel = dangerLevel;
```

Comment:
You shouldn't need these reassignments- you should be able to use the params as written in the parens.